picoquic/picoquic_internal.h

```diff
@@ -79,7 +79,8 @@ typedef enum {
     picoquic_frame_type_ack_range_min_old = 0xa0,
     picoquic_frame_type_ack_range_max_old = 0xbf,
     picoquic_frame_type_stream_range_min_old = 0xc0,
-    picoquic_frame_type_stream_range_max_old = 0xcf
+    picoquic_frame_type_stream_range_max_old = 0xcf,
+    picoquic_frame_type_expired_stream_data = 0xf0
```

Comment:
Is picoquic_frame_type_expired_stream_data the expansion type?

Reply:
I should had left this out of the PR.